### PR TITLE
feat: resolve panic cause by importMeta type

### PR DESCRIPTION
**Pull request by sunrabbit123**
**Description:**

```ts
declare global { interface ImportMeta {foo?: () => void} };

if (import.meta.foo) {
  import.meta.foo();
}
```

**BREAKING CHANGE:**

**Related issue (if exists):**

### CODE REVIEW

File: crates/stc_ts_file_analyzer/src/analyzer/expr/mod.rs [hunk 1]
```diff
@@ -333,15 +333,6 @@ impl Analyzer<'_, '_> {
         ty.freeze();
         self.cur_facts.assert_clone_cheap();
 
-        if !self.config.is_builtin && !ty.is_any() {
```
Comment (kdy1) [inline crates/stc_ts_file_analyzer/src/analyzer/expr/mod.rs:336]: This should not be removed.

Comment (kdy1) [reply crates/stc_ts_file_analyzer/src/analyzer/expr/mod.rs:336]: Please change the span of the generated `Type` instead

**Review by kdy1 — CHANGES_REQUESTED** [review]

File: crates/stc_ts_file_analyzer/src/analyzer/expr/mod.rs (hunk 1 above)
Comment (sunrabbit123) [reply crates/stc_ts_file_analyzer/src/analyzer/expr/mod.rs:336]: span of builtin type eqaul dummy_span
chnage to span of `import.meta` keyword?
@kdy1

Comment (kdy1) [reply crates/stc_ts_file_analyzer/src/analyzer/expr/mod.rs:336]: Yes

**Review by kdy1 — APPROVED** [review]

**Pull request merged**